coalib/bearlib/languages/definitions/PowerShell.py

```diff
@@ -0,0 +1,11 @@
+from coalib.bearlib.languages.Language import Language
+
+
+@Language
+class PowerShell:
+    extensions = '.ps1',
+    comment_delimiter = '#'
+    multiline_comment_delimiters = {'<#': '#>'}
+    string_delimiters = {'"': '"', "'": "'"}
+    multiline_string_delimiters = {'"': '"', "'": "'"}
```

Comment:
can you show me some info about these being multiline ?

Reply:
Sure, I found some info at https://www.sconstantinou.com/powershell-quotes/

Reply:
Great.
Then you can do `multiline_string_delimiters = string_delimiters` to make it more clear they are identical.